core-commands.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
 			let [target, amount] = message.split(',');
 			target = utils.toId(target);
 			amount = parseInt(amount);
-			if (!target || !amount || isNaN(amount)) return this.send('Syntax: ``.addquills username, amount``');
+			if (!target || !amount || isNaN(amount)) return this.send('Syntax: ``' + Config.commandToken + 'addquills username, amount``');
```

Comment:
This should use a template string instead:
```js
this.send(`Syntax: \`\`${config.commandToken}addquills username, amount\`\``);
```

Reply:
Oh okay, sorry I didn't know the string was different.